src/py_proc.c

```diff
@@ -672,7 +673,13 @@ py_proc__start(py_proc_t * self, const char * exec, char * argv[]) {
   #else                                                               /* UNIX */
   self->pid = fork();
   if (self->pid == 0) {
+    // If we are not writing to file we need to ensure the child process is
+    // not writing to stdout.
+    if (output_file == NULL)
+      close(STDOUT_FILENO);
```

Comment:
This leave the process without any stdout file descriptor which is really unusual. Every Unix program expects to have something in stdout/stderr/stdin:

```sh
$ ./src/austin echo test
echo: write error: Bad file descriptor
```

It'd be better to open `/dev/null` and dup to it stdout.

Reply:
Fixed

Reply:
Fixed

Reply:
Fixed